src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -361,7 +362,7 @@ void EnSth_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Ve
 
     if (limbIndex == 15) {
         Matrix_MultVec3f(&D_80B0B49C, &this->actor.focus.pos);
-        if (this->actor.params != 0) {
+        if (this->actor.params != 0) { //Children 
```

Comment:
```suggestion
        if (this->actor.params != 0) { // Children 
```

Reply:
this shows the formatter probably needs to be run